l10n_th_amount_to_text/models/res_currency.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2020 Ecosoft Co., Ltd. (http://ecosoft.co.th)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+import logging
+
+from odoo import models
+
+_logger = logging.getLogger(__name__)
+
+try:
+    from num2words import num2words
+except ImportError:
+    _logger.warning(
+        "The num2words python library is not installed,"
+        "amount-to-text features won't be fully available."
+    )
+    num2words = None
```

Comment:
This code is the same as the odoo core.
https://github.com/odoo/odoo/blob/48f3fa1673cabaa6b4031fbe524be632d3897ec0/odoo/addons/base/models/res_currency.py#L14